.vscode/settings.json

```diff
@@ -7,4 +7,13 @@
     // Makes IntelliSense work with Flow.  See
     // https://github.com/Microsoft/vscode-react-native/blob/master/doc/intellisense.md .
     "flow.useNPMPackagedFlow": true,
+    
+    // Enable EslintIntegration
+    // https://github.com/prettier/prettier-eslint#related
+    "prettier.eslintIntegration": true,
+    
+    "prettier.parser": "flow",
+    "prettier.trailingComma": "all",
+    "prettier.singleQuote": true,
+    "prettier.printWidth": 100,
```

Comment:
What's the purpose of these last four lines? The extension is supposed to read this configuration from `.prettierrc`. Is that not working for some reason?

Reply:
It was not working so I added them here.
Also reported at https://github.com/prettier/prettier-vscode/issues/206#issuecomment-334994832
https://github.com/prettier/prettier-vscode/issues/206#issuecomment-383556450

Reply:
Hmm. I don't reproduce this issue -- these all work for me without repeating these lines here.

It looks like that comment was on an unrelated issue, and never got filed as its own issue (or if it did, it didn't link to this one). That makes it easy to get dropped.

Want to file an issue on prettier-vscode describing the issue as you're seeing it? That would help it get resolved there.

I think you can work around the issue for yourself by putting these in your user settings. If not (or if it turns out several of us hit this issue), I'd be happy to include these here as a workaround. If we do, though, it's important to have a comment explaining why -- otherwise, because when a reader looks at the docs they see that these *shouldn't* be necessary, they're confusing.

Separately, when some settings like this get duplicated in two places, it's important to have a comment in each place saying so. Otherwise, they're very likely to diverge as people update one and don't see the other.